compiler/src/iree/compiler/Codegen/LLVMCPU/DispatchABI.cpp

```diff
@@ -765,9 +763,8 @@ MemRefDescriptor HALDispatchABI::loadBinding(Operation *forOp, int64_t ordinal,
   // place to do it.
 
   // Cast to the desired memref element type.
-  auto elementType = typeConverter->convertType(memRefType.getElementType());
   Value typedPtrValue = builder.create<LLVM::BitcastOp>(
-      loc, LLVM::LLVMPointerType::get(elementType), basePtrValue);
+      loc, LLVM::LLVMPointerType::get(context), basePtrValue);
```

Comment:
Same here. Drop the use of the `BitCastOp` and just forward to source value to all the users.

Reply:
This is done. Marking as resolved!